### ISSUE TITLE
[BUG] Projectile find file returns ^Gfd: no such device or address

### ISSUE THREAD

**Issue by t-augusto** [hlissner/doom-emacs#1931]
After running doom update, my projectile-find-file returns the error message in the title of this issue.

![no_such_device](https://user-images.githubusercontent.com/24945937/67292156-06b28280-f4b9-11e9-8f5c-fc2f3af24632.png)

This happens on every project I have on my machine, be it local or over tramp.
Projectile now only sees the file I had already opened/cached, and does not search any other files in the project.

**System information**

```
((emacs
  (version . "26.3")
  (features . "XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM THREADS LIBSYSTEMD LCMS2")
  (build . "Oct 16, 2019")
  (buildopts "--prefix= --prefix=/snap/emacs/current/usr 'CFLAGS= -I/build/emacs/parts/emacs/install/usr/include' 'CPPFLAGS= -I/build/emacs/parts/emacs/install/usr/include' 'LDFLAGS= -L/build/emacs/parts/emacs/install/lib -L/build/emacs/parts/emacs/install/usr/lib -L/build/emacs/parts/emacs/install/lib/x86_64-linux-gnu -L/build/emacs/parts/emacs/install/usr/lib/x86_64-linux-gnu'")
  (windowsys . batch)
  (daemonp . server-running))
 (doom
  (version . "2.0.9")
  (build . "HEAD -> develop, origin/develop, origin/HEAD 0ccf31635 2019-10-22 01:58:17 -0400"))
 (system
  (type . gnu/linux)
  (config . "x86_64-pc-linux-gnu")
  (shell . "/usr/bin/fish")
  (uname . "Linux 5.0.0-31-generic #33~18.04.1-Ubuntu SMP Tue Oct 1 10:20:39 UTC 2019 x86_64")
  (path "~/bin" "/usr/bin" "/bin" "/usr/sbin" "/sbin" "/usr/local/bin" "/snap/bin" "/snap/emacs/current/usr/libexec/emacs/26.3/x86_64-pc-linux-gnu"))
 (config
  (envfile . envvar-file)
  (elc-files . 31)
  (modules :completion company helm :ui doom doom-dashboard doom-quit hl-todo indent-guides modeline (popup +all +defaults) :editor (evil +everywhere) file-templates fold multiple-cursors snippets :emacs dired :term eshell :tools editorconfig eval flycheck magit make :lang cc data emacs-lisp (haskell +intero) markdown python sh :config (default +bindings +smartparens))
  (packages "n/a")
  (elpa "n/a")))
```

Steps to reproduce
1. I only ran doom refresh and update after updating my doom config to add magit to my installed packages.
2. SPC SPC which is bound to projectile-find-file does not search the current project

**Comment by hlissner** [comment]
Odd. I cannot reproduce this. 

1. What is the value of `projectile-git-command`? (`SPC h v projectile-git-command` will tell you).
2. It appears you have byte-compiled your config. Does your issue persist after you run `bin/doom clean` and restart Emacs?
3. What does `fd --version` output on the command line?

**Comment by t-augusto** [comment]
1. ``` 
projectile-git-command is a variable defined in projectile.el.

Value
"fd . --color=never --type f -0 -H -E .git"

Original Value
"git ls-files -zco --exclude-standard"
```
2. Issue persists even after cleaning.

3.It seems my FD does not support --version. I have the fdclone package installed on ubuntu 18.04
Package: fdclone
Version: 3.01b-1build2
Priority: optional
Section: universe/shells

**Comment by t-augusto** [comment]
Ok, I've updated my doom emacs, via doom upgrade. Removed wrong fd package from my system and installed the correct one.
However i still get the same error on my projects.
projectile-git-command is the fd command I sent in the comments above.
fd --version is `fd 7.4.0`

I decided to check if it was happening on specific projects or something:
I have 5 git repos, 1 is mounted over sshfs and the rest are all local.
On the one mounted over sshfs I receive the same error, of no such device or address.
On the local ones, 2 of them projectile-find-file works correctly, and searches the whole project.
On the other 2 however, I still have the same error as before.
